dspace/config/default.license

```diff
@@ -3,34 +3,16 @@ This sample license is provided for informational purposes only.
 
 NON-EXCLUSIVE DISTRIBUTION LICENSE
 
-By signing and submitting this license, you (the author(s) or copyright
-owner) grants to DSpace University (DSU) the non-exclusive right to reproduce,
-translate (as defined below), and/or distribute your submission (including
-the abstract) worldwide in print and electronic format and in any medium,
-including but not limited to audio or video.
-
-You agree that DSU may, without changing the content, translate the
-submission to any medium or format for the purpose of preservation.
-
-You also agree that DSU may keep more than one copy of this submission for
-purposes of security, back-up and preservation.
-
-You represent that the submission is your original work, and that you have
-the right to grant the rights contained in this license. You also represent
-that your submission does not, to the best of your knowledge, infringe upon
-anyone's copyright.
-
-If the submission contains material for which you do not hold copyright,
-you represent that you have obtained the unrestricted permission of the
-copyright owner to grant DSU the rights required by this license, and that
-such third-party owned material is clearly identified and acknowledged
-within the text or content of the submission.
-
-IF THE SUBMISSION IS BASED UPON WORK THAT HAS BEEN SPONSORED OR SUPPORTED
-BY AN AGENCY OR ORGANIZATION OTHER THAN DSU, YOU REPRESENT THAT YOU HAVE
-FULFILLED ANY RIGHT OF REVIEW OR OTHER OBLIGATIONS REQUIRED BY SUCH
-CONTRACT OR AGREEMENT.
-
-DSU will clearly identify your name(s) as the author(s) or owner(s) of the
-submission, and will not make any alteration, other than as allowed by this
-license, to your submission.
+By signing and submitting this license, you (the author(s) or copyright owner) grants to DSpace University (DSU) the non-exclusive right to reproduce, translate (as defined below), and/or distribute your submission (including the abstract) worldwide in print and electronic format and in any medium, including but not limited to audio or video.
```

Comment:
@alexandrevryghem : Why did you modify the `default.license` in this PR?  It seems to be an unrelated change?  I suspect this was accidental and needs to be reverted.

Reply:
I did it on purpose, but I can revert it 😅. In PR [#2229](https://github.com/DSpace/dspace-angular/pull/2229) the way the `default.license` was rendered has been changed to also display new lines. The problem is that because we added manual new line breaks in that file, those line breaks are also being displayed. This results in the lines not wrapping correctly on some screens. I can move this to a different PR if you prefer, but I thought that the change was small enough to add it to this PR 🤷‍♂️
![image](https://github.com/DSpace/DSpace/assets/43750381/7d0e4f8c-5253-44af-9064-8c2c9a2a435e)
![image](https://github.com/DSpace/DSpace/assets/43750381/2c513d8f-424f-43dc-ab8d-c11fe8eecf14)